src/website/views/components/data-table/_examples.pug

```diff
@@ -0,0 +1,5827 @@
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-chevron-left" aria-hidden="true"></i>
+                    </div>
+                  </button>
+                  <button class="chi-button -flat" aria-label="Page 1">1</button>
+                  <button class="chi-button -flat" aria-label="Page 2">2</button>
+                  <button class="chi-button -flat -active" aria-label="Page 3">3</button>
+                  <button class="chi-button -flat" aria-label="Page 4">4</button>
+                  <button class="chi-button -flat" aria-label="Page 5">5</button>
+                  <div class="chi-button -flat" aria-hidden="true" disabled>...</div>
+                  <button class="chi-button -flat" aria-label="Page 12">12</button>
+                  <button class="chi-button -icon -flat" aria-label="Next page">
+                    <div class="chi-button__content">
+                      <i class="chi-icon icon-chevron-right" aria-hidden="true"></i>
+                    </div>
+                  </button>
+                </div>
+              </div>
+              <div class="chi-pagination__end">
+                <div class="chi-pagination__jumper">
+                  <label class="chi-pagination__label" for="jumper-input-8">Go to page:</label>
+                  <input class="chi-input" type="text" id="jumper-input-8" />
+                </div>
+              </div>
+            </div>
+          </nav>
+        </div>
+      </div>
+
+h3 Accordion Expanded
+.example.-mb--3
+  .-p--3
+    .chi-data-table
+      .chi-data-table__head
+        .chi-data-table__row
+          .chi-data-table__cell.-expandable
+          .chi-data-table__cell
+            div Name
+          .chi-data-table__cell
+            div ID
+          .chi-data-table__cell
+            div Last Login
+      .chi-data-table__body
+        .chi-data-table__row
+          .chi-data-table__cell.-expandable
+            button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+              .chi-button__content
+                i.chi-icon.icon-chevron-right
+          .chi-data-table__cell(data-label="Name")
+            div Name 1
+          .chi-data-table__cell(data-label="ID")
+            div name-1
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 3:26 PM    
+        .chi-data-table__row.-expanded
+          .chi-data-table__cell.-expandable
+            button.chi-button.-icon.-flat.-sm(aria-label='Expand row')
+              .chi-button__content
+                i.chi-icon.icon-chevron-right
+          .chi-data-table__cell(data-label="Name")
+            div Name 2
+          .chi-data-table__cell(data-label="ID")
+            div name-2
+          .chi-data-table__cell(data-label="Last Login")
+            div Dec 18, 2020 2:38 AM    
+        .chi-data-table__row-child
+            div Some random content
```

Comment:
Can we change all instances of 'Some random content' to 'Accordion content'?

We may also want to look into the styling of this as it isn't rendering well on mobile.
```suggestion
            div Accordion content
```